cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help

```diff
@@ -43,6 +43,9 @@ OPTIONS
        --debug
            Run in remote debugging mode.
```

Comment:
```suggestion
           Run in remote the debugging mode.
```

Reply:
I'm not sure if this change is correct. "remote debugging" is a thing.

Reply:
Created Issue #24275

Reply:
> I'm not sure if this change is correct. "remote debugging" is a thing.

Yeah I also think that this should better be kept as it is. +1 for the existing way.